[2022-04-13 22:36:04,038] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:21+00:00 [queued]>
[2022-04-13 22:36:04,095] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:21+00:00 [queued]>
[2022-04-13 22:36:04,096] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:04,096] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:36:04,097] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:04,194] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-02 06:00:21+00:00
[2022-04-13 22:36:04,215] {standard_task_runner.py:52} INFO - Started process 11266 to run task
[2022-04-13 22:36:04,235] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:21+00:00', '--job-id', '1941', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpo0d1pnjy', '--error-file', '/tmp/tmphw71i4v3']
[2022-04-13 22:36:04,253] {standard_task_runner.py:77} INFO - Job 1941: Subtask ingest
[2022-04-13 22:36:04,719] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:21+00:00 [running]> on host 13db72185839
[2022-04-13 22:36:05,026] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:21+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:21+00:00
[2022-04-13 22:36:05,030] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:36:05,032] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:36:05,050] {subprocess.py:85} INFO - Output:
[2022-04-13 22:36:05,064] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:36:05,065] {subprocess.py:89} INFO - dags
[2022-04-13 22:36:05,067] {subprocess.py:89} INFO - logs
[2022-04-13 22:36:05,069] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:36:05,069] {subprocess.py:89} INFO - output_2022-03.csv
[2022-04-13 22:36:05,078] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:36:05,081] {subprocess.py:89} INFO - plugins
[2022-04-13 22:36:05,082] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:36:05,083] {subprocess.py:89} INFO - scripts
[2022-04-13 22:36:05,085] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:36:05,086] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:36:05,360] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060021, start_date=20220413T223604, end_date=20220413T223605
[2022-04-13 22:36:05,453] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:36:05,670] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
